File: TRNMNT/ClientApp/app/core/model/weight-division.model.ts
export class WeightDivisionModel {
    weightDivisionId: AAGUID;
    name: string;
    weight: number;
    description: string;
    categoryId: string;

    /**
     *
     */
    constructor(name: string, id?: AAGUID ) { 
        this.weightDivisionId = id;
        this.name = name;
    }
}

File: TRNMNT.Core/Model/WeightDivisionModel.cs
using System;

namespace TRNMNT.Core.Model
{
    public class WeightDivisionModel
    {
        public Guid WeightDivisionId { get; set; }
        public string Name { get; set; }
        public int Weight { get; set; }
        public string Descritpion { get; set; }
        public string CategoryId { get; set; }
    }
}
